Application/Extensions/KeyStageExtensions.cs

```diff
@@ -6,12 +6,7 @@ public static class KeyStageExtensions
 {
     public static string DisplayList(this IEnumerable<KeyStage> keyStages)
     {
-        var commaSeparated = string.Join(", ", keyStages.Select(e => e.Id).Distinct().OrderBy(e => e));
-
-        var lastCommaIndex = commaSeparated.LastIndexOf(",", StringComparison.Ordinal);
-
-        if (lastCommaIndex == -1) return commaSeparated;
-
-        return commaSeparated.Remove(lastCommaIndex, 1).Insert(lastCommaIndex, " and");
+        var keyStageStrings = keyStages.Select(e => e.Id.ToString()).Distinct().OrderBy(e => e);
```

Comment:
Should we consider better naming for this method?

Reply:
Not sure this is really worth picking up, but have removed the variable anyway.